Tests/TestingApps/PurchaseTesterSwiftUI/fastlane/Fastfile

```diff
@@ -0,0 +1 @@
+import("../../../../fastlane/Fastfile")
```

Comment:
With this I guess the `PurchaseTesterSwiftUI` project will have access to all the project fastfile's functions... There are probably many that shouldn't be called from here, but then again, I think it should be ok.

Reply:
Yeah it's fine. It's the same approach we use in the installation tests.